### PR TITLE
Use Finality Tag for Astar

### PR DESCRIPTION
**Pull request by flodesi**
## Motivation
following the [cherry pick for astar custom finaility from chainlink repo](https://github.com/smartcontractkit/ccip/pull/1324), need to enable it for astar

### CODE REVIEW

**Review by friedemannf — APPROVED** [review]

**Comment by friedemannf** [comment]
You'll need to run `make config-docs` to re-generate the documentation

File: core/chains/evm/config/toml/defaults/Astar_Mainnet.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ChainID = '592'
 # FinalityTag will be enabled once the custom tag is implemented
```
Comment (finleydecker) [inline core/chains/evm/config/toml/defaults/Astar_Mainnet.toml]:
```suggestion
```

Comment (finleydecker) [reply core/chains/evm/config/toml/defaults/Astar_Mainnet.toml]: We can remove this comment now :)

File: core/chains/evm/config/toml/defaults/Astar_Shibuya.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ChainID = '81'
 # FinalityTag will be enabled once the custom tag is implemented
```
Comment (finleydecker) [inline core/chains/evm/config/toml/defaults/Astar_Shibuya.toml]:
```suggestion
```

**Review by finleydecker — CHANGES_REQUESTED** [review]

**Review by finleydecker — APPROVED** [review]

**Review by matYang — APPROVED** [review]

**Pull request merged**